src/storage/memory.ts

```diff
@@ -67,6 +69,17 @@ export function buildMemoryStorage(
         }
       }
 
+      /* istanbul ignore if 'only available on super recent browsers' */
+      if (cloneData === 'double' && value !== undefined) {
```

Comment:
```suggestion
      if (cloneData === 'double') {
```

value will never get undefined here